### PR TITLE
Updated configuring realms chapter #1483

### PR DESCRIPTION
**Pull request by agagancarczyk**
Updated configuring realms chapter #1483

### CODE REVIEW

**Comment by agagancarczyk** [comment]
@andymunro This branch has only my changes. Hope this is ok now. Thanks

File: server_admin/topics/login-settings/forgot-password.adoc [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+ifeval::[{project_product}==true]
 . Click *Realm Settings* in the menu.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Realm settings* in the menu.
+endif::[]
 . Click the *Login* tab.
 +
 .Login tab
 image:{project_images}/login-tab.png[Login Tab]
```
Comment (andymunro) [inline server_admin/topics/login-settings/forgot-password.adoc:16]: This image is the old version.

File: server_admin/topics/login-settings/forgot-password.adoc [hunk 2]
```diff
@@ -37,5 +52,10 @@ To change this behavior, perform these steps:
 image:{project_images}/reset-credentials-flow.png[Reset Credentials Flow]
 +
 If you do not want to reset the OTP, set the `Reset OTP` requirement to *Disabled*.
+ifeval::[{project_product}==true]
 . Click the *Required Actions* tab. Ensure _Update Password_ is enabled.
-
+endif::[]
+ifeval::[{project_community}==true]
+. Navigate back to Authentication and click the *Required actions* tab. Ensure _Update Password_ is enabled.
+image:{project_images}/reset-credentials-required-actions.png[Reset Credentials Required Actions]
```
Comment (andymunro) [inline server_admin/topics/login-settings/forgot-password.adoc]:
```suggestion
. Click *Authentication* in the menu.
. Click the *Required actions* tab. 
. Ensure *Update Password* is enabled.
+
.Required Actions
image:{project_images}/reset-credentials-required-actions.png[Required Actions]
```

File: server_admin/topics/account.adoc [hunk 3]
```diff
@@ -2,12 +2,12 @@
 
 == Account Console
```
Comment (andymunro) [inline server_admin/topics/account.adoc]: This file was already merged as part of a different PR.

File: server_admin/topics/realms/keys.adoc [hunk 4]
```diff
@@ -42,9 +48,19 @@ Rotating keys also applies to offline tokens. To make sure they are updated, the
 . Click *Realm settings*.
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: server_admin/topics/realms/keys.adoc [hunk 5]
```diff
@@ -14,8 +14,14 @@ When a realm is created a key pair and a self-signed certificate is automaticall
 . Select the realm in the Admin Console.
 . Click *Realm settings*.
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: server_admin/topics/realms/keys.adoc [hunk 6]
```diff
@@ -14,8 +14,14 @@ When a realm is created a key pair and a self-signed certificate is automaticall
 . Select the realm in the Admin Console.
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]: This step is not needed.

File: server_admin/topics/realms/keys.adoc [hunk 7]
```diff
@@ -83,7 +112,12 @@ For the associated certificate chain to be loaded it must be imported to the Jav
 . Click *Realm settings*.
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: server_admin/topics/realms/keys.adoc [hunk 8]
```diff
@@ -97,9 +131,19 @@ For the associated certificate chain to be loaded it must be imported to the Jav
 . Select the realm in the Admin Console.
 . Click Realm settings.
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: server_admin/topics/realms/keys.adoc [hunk 9]
```diff
@@ -108,9 +152,19 @@ For the associated certificate chain to be loaded it must be imported to the Jav
...
```
Comment (andymunro) [inline server_admin/topics/realms/keys.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: getting_started/topics/standalone/proc-logging-in-admin-console.adoc [hunk 10]
```diff
@@ -23,7 +23,7 @@ image:images/admin-login.png[Admin console login screen]
 The initial screen for the admin console appears.
 +
 .Admin console
-image:images/admin-console.png[Admin console]
+image:{project_images}/admin-console.png[Admin Console]
```
Comment (andymunro) [inline getting_started/topics/standalone/proc-logging-in-admin-console.adoc]: This image applies to the new Admin Console, but the version under rhsso-images is showing the same image.  It should be unchanged for rhsso-images.

File: server_admin/topics/realms/proc-creating-a-realm.adoc [hunk 11]
```diff
@@ -13,10 +13,15 @@ realm and only be able to interact with customer-facing apps.
...
 . Click *Add Realm*.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Create Realm*.
+endif::[]
 +
-.Add realm menu
-image:{project_images}/add-realm-menu.png[Add realm menu]
+.Create realm menu
+image:{project_images}/create-realm-menu.png[Create realm menu]
```
Comment (andymunro) [inline server_admin/topics/realms/proc-creating-a-realm.adoc]: This image applies to the new Admin Console, but the version under rhsso-images is showing the same image. It should be unchanged for rhsso-images.

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks better, @agagancarczyk. Just a few changes needed.

File: server_admin/topics/login-settings/forgot-password.adoc (hunk 1 above)
Comment (agagancarczyk) [reply server_admin/topics/login-settings/forgot-password.adoc:16]: Thanks @andymunro , I have updated the image.

File: server_admin/topics/account.adoc (hunk 3 above)
Comment (agagancarczyk) [reply server_admin/topics/account.adoc]: No problem, I have restored it back.

File: getting_started/topics/standalone/proc-logging-in-admin-console.adoc (hunk 10 above)
Comment (agagancarczyk) [reply getting_started/topics/standalone/proc-logging-in-admin-console.adoc]: Updated this, but please double-check if this is correct now. I may have misunderstood the directions.

File: server_admin/topics/realms/proc-creating-a-realm.adoc (hunk 11 above)
Comment (agagancarczyk) [reply server_admin/topics/realms/proc-creating-a-realm.adoc]: Updated this, but please double-check if this is correct now. I may have misunderstood the directions.

File: server_admin/topics/realms/ssl.adoc [hunk 12]
```diff
@@ -8,21 +8,46 @@ Browsers and applications that interact with the realm honor the SSL/HTTPS requi
...
 
 .Procedure 
-. Click *Realm Settings* in the menu.
 
+ifeval::[{project_product}==true]
+. Click *Realm Settings* in the menu.
 . Click the *Login* tab.
 +
 .Login tab
 image:{project_images}/login-tab.png[Login tab]
```
Comment (andymunro) [inline server_admin/topics/realms/ssl.adoc:17]: This screen shot is for the new console, but it also appears in the old console documentation.

File: server_admin/topics/login-settings/forgot-password.adoc (hunk 1 above)
Comment (andymunro) [inline server_admin/topics/login-settings/forgot-password.adoc:16]: This screen shot is for the new console, but it also appears in the old console documentation.

File: server_admin/topics/login-settings/remember-me.adoc [hunk 13]
```diff
@@ -4,14 +4,29 @@
...
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Realm settings* in the menu.
 . Click the *Login* tab.
-. Toggle the *Remember Me* switch to *ON*.
+endif::[]
+. Toggle the *Remember Me* switch to *On*.
 +
 .Login tab
+ifeval::[{project_product}==true]
```
Comment (andymunro) [inline server_admin/topics/login-settings/remember-me.adoc:17]: This screen shot is for the new console, but it also appears in the old console documentation.

File: server_admin/topics/realms/proc-creating-a-realm.adoc [hunk 14]
```diff
@@ -13,7 +13,12 @@ realm and only be able to interact with customer-facing apps.
...
 
+ifeval::[{project_product}==true]
 . Click *Add Realm*.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Create Realm*.
+endif::[]
 +
 .Add realm menu
 image:{project_images}/add-realm-menu.png[Add realm menu]
```
Comment (andymunro) [inline server_admin/topics/realms/proc-creating-a-realm.adoc:24]:
```suggestion
image:{project_images}/create-realm-menu.png[Add realm menu]
```

Comment (andymunro) [inline server_admin/topics/realms/proc-creating-a-realm.adoc:24]: Sorry, I see that the original file was actually add-realm-menu.png.  So here is what we need.

- create-realm-menu.png under Keycloak-images should be renamed  to add-realm-menu.png
- The existing add-realm-menu.png under rhsso-images is working. You don't need a create-realm-menu.png there. You can delete rhsso-images/create-realm-menu.png.

Comment (andymunro) [reply server_admin/topics/realms/proc-creating-a-realm.adoc:24]: Also, the existing add-realm-menu.png under rhsso-images is working. You don't need a create-realm-menu.png there.

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @agagancarczyk.  Looking good.  Just a few final changes that show up when I build the keycloak or SSO versions of the book.

File: server_admin/topics/realms/proc-creating-a-realm.adoc (hunk 14 above)
Comment (agagancarczyk) [reply server_admin/topics/realms/proc-creating-a-realm.adoc:24]: Thanks @andymunro , this makes sense.

**Comment by agagancarczyk** [comment]
> Thanks, @agagancarczyk. Looking good. Just a few final changes that show up when I build the keycloak or SSO versions of the book.

Thanks @andymunro I have made the requested changes. Hope this looks good now :)

**Review by andymunro — APPROVED** [review]
Okay, thanks for the corrections, @agagancarczyk. Looks great.
@ssilvert Please merge.

**Comment by ssilvert** [comment]
@agagancarczyk Please fix conflicts.

**Comment by agagancarczyk** [comment]
This is fixed now @ssilvert . Thanks

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**